tfx_addons/feast_examplegen/example_usage.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Losely based on internal examples and Feast tutorial: https://docs.feast.dev/v/v0.6-branch/user-guide/feature-retrieval"""
+
+import tfx.v1 as tfx
+from feast.infra.offline_stores.bigquery import BigQueryOfflineStoreConfig
+from feast.repo_config import RepoConfig
+
+from tfx_addons.feast_examplegen import FeastExampleGen
+
+# This is a pydantic model, so it can be serialized into JSON in the component and reloaded back into the Python object in the executor!
+repo_config = RepoConfig(
```

Comment:
I am thinking we should have a `FeastConnConfig` object encapsulate `RepoConfig` and `project` (and possibly other fields) for consistency with other ExampleGens

Reply:
Mmm sure. I would however use pydantic here instead of using protobuf. Given RepoConfig is already a pydantic model, we can serialize it to JSON and then back from JSON easily

Reply:
Updated!